src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,12 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Integer> {
+
+    @Query("SELECT b FROM Book b WHERE LOWER(b.author) = LOWER(:author)")
+    Book bookByAuthor(@Param("author") String author);
```

Comment:
Two things here:

1. It's better return the result wrapped in an `Optional`. Thus, we can avoid null checks
2. `@Query` is fine, but we should use for complexies queries or when [Query methods](https://docs.spring.io/spring-data/data-jpa/docs/current/reference/html/#repositories.query-methods) doesn't support the keyword one wants to use.
Change to use query methods.